containers/sarek/environment.yml

```diff
@@ -0,0 +1,24 @@
+# You can use this file to create a conda environment for this pipeline:
+# conda env create -f environment.yml
+name: sarek-core
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+
+dependencies:
+  - bcftools=1.8
+  - bwa=0.7.17
+  - conda-forge::openjdk=8.0.144 # Needed for FastQC docker - see bioconda/bioconda-recipes#5026
+  - fastqc=0.11.7
+  - freebayes=1.2.0
+  - gatk4=4.0.3.0
```

Comment:
You can use the 4.0.4.0, the executable is back to being `gatk` and not `gatck-launch` anymore

Reply:
Fine, will change the name in processes as well. In fact we have 4.0.6.0 also

Reply:
Even better ;-)